public/locales/en.json

```diff
@@ -446,5 +446,51 @@
     "adminForEvents": "Admin for events",
     "addedAsAdmin": "User is added as admin.",
     "talawaApiUnavailable": "Talawa-API service is unavailable. Is it running? Check your network connectivity too."
+  },
+  "userNavbar": {
+    "talawa_portal": "Talawa",
+    "home": "Home",
+    "people": "People",
+    "events": "Events",
+    "chat": "Chat",
+    "donate": "Donate",
+    "myTasks": "My Tasks",
+    "settings": "Settings",
+    "language": "Language",
+    "logout": "Logout",
+    "close": "Close"
+  },
+  "userLogin": {
+    "login": "Login",
+    "forgotPassword": "Forgot Password?",
+    "loginIntoYourAccount": "Login into your account",
+    "emailAddress": "Email Address",
+    "enterEmail": "Enter your email address",
+    "password": "Password",
+    "enterPassword": "Enter your password",
+    "register": "Register",
+    "invalidDetailsMessage": "Please enter a valid email and password.",
+    "notAuthorised": "Sorry! you are not Authorised!",
+    "invalidCredentials": "Entered credentials are incorrect. Please enter valid credentials.",
+    "talawaApiUnavailable": "Talawa-API service is unavailable. Is it running? Check your network connectivity too."
+  },
+  "userRegister": {
+    "register": "Register",
+    "firstName": "First Name",
+    "enterFirstName": "Enter your first name",
+    "lastName": "Last Name",
+    "enterLastName": "Enter your last name",
+    "emailAddress": "Email Address",
+    "enterEmail": "Enter your email address",
+    "password": "Password",
+    "enterPassword": "Enter your password",
+    "confirmPassword": "Confirm Password",
+    "enterConfirmPassword": "Enter your password to confirm",
+    "alreadyhaveAnAccount": "Already have an account?",
+    "login": "Login",
+    "afterRegister": "Successfully registered. Please wait for admin to approve your request.",
+    "passwordNotMatch": "Confirm password and password doesn't match.",
+    "invalidDetailsMessage": "Please enter valid details.",
+    "talawaApiUnavailable": "Talawa-API service is unavailable. Is it running? Check your network connectivity too."
```

Comment:
In my opinion we can phrase this in a little better manner because both the message leave the end user with little dilemma with the current one. @evasharma12 can you help out on this.

Reply:
@evasharma12 Please provide me the appropriate messages, I'll update them.

Reply:
@beingnoble03 I find the passwordNotMatch and talawaApiUnavailable texts improvable. You can phrase them like this:
"passwordNotMatch": "Password doesn't match. Confirm Password and try again."
"talawaApiUnavailable": "Talawa-API service is unavailable. Kindly check your network connection and wait for a while."